Spigot-Server-Patches/0496-Fix-commodore.patch

```diff
@@ -0,0 +1,1300 @@
+-                                </relocation>
+-                                <relocation>
+-                                    <pattern>org.bukkit.craftbukkit</pattern>
+-                                    <shadedPattern>org.bukkit.craftbukkit.v${minecraft_version}</shadedPattern>
+-                                    <excludes>
+-                                        <exclude>org.bukkit.craftbukkit.Main*</exclude>
+-                                    </excludes>
+-                                </relocation>
+-                                <relocation>
+-                                    <pattern>net.minecraft.server</pattern>
+-                                    <shadedPattern>net.minecraft.server.v${minecraft_version}</shadedPattern>
+-                                </relocation>
++<!--                                <relocation>-->
++<!--                                    <pattern>org.apache.commons.codec</pattern>-->
++<!--                                    <shadedPattern>org.bukkit.craftbukkit.libs.org.apache.commons.codec</shadedPattern>-->
++<!--                                </relocation>-->
++<!--                                <relocation>-->
++<!--                                    <pattern>org.apache.commons.io</pattern>-->
++<!--                                    <shadedPattern>org.bukkit.craftbukkit.libs.org.apache.commons.io</shadedPattern>-->
++<!--                                </relocation>-->
++<!--                                <relocation>-->
++<!--                                    <pattern>org.apache.commons.lang3</pattern>-->
++<!--                                    <shadedPattern>org.bukkit.craftbukkit.libs.org.apache.commons.lang3</shadedPattern>-->
++<!--                                </relocation>-->
++<!--                                <relocation>-->
++<!--                                    <pattern>org.objectweb.asm</pattern>-->
++<!--                                    <shadedPattern>org.bukkit.craftbukkit.libs.org.objectweb.asm</shadedPattern>-->
++<!--                                </relocation>-->
++<!--                                <relocation>-->
++<!--                                    <pattern>org.bukkit.craftbukkit</pattern>-->
++<!--                                    <shadedPattern>org.bukkit.craftbukkit.v${minecraft_version}</shadedPattern>-->
++<!--                                    <excludes>-->
++<!--                                        <exclude>org.bukkit.craftbukkit.Main*</exclude>-->
++<!--                                    </excludes>-->
++<!--                                </relocation>-->
++<!--                                <relocation>-->
++<!--                                    <pattern>net.minecraft.server</pattern>-->
++<!--                                    <shadedPattern>net.minecraft.server.v${minecraft_version}</shadedPattern>-->
++<!--                                </relocation>-->
```

Comment:
Oopsie woopsie? I'm pretty sure relocations of the packages are on purpose, at least the NMS & CraftBukkit ones.